Python/optimizer.c

```diff
@@ -154,13 +154,16 @@ PyUnstable_GetOptimizer(void)
 }
 
 static _PyExecutorObject *
-make_executor_from_uops(_PyUOpInstruction *buffer, const _PyBloomFilter *dependencies);
+make_executor_from_uops(_PyUOpInstruction *buffer, int length, const _PyBloomFilter *dependencies);
 
 static int
 init_cold_exit_executor(_PyExecutorObject *executor, int oparg);
 
+/* The maximum number of exits in a trace cannot reach 1/4 of its length */
```

Comment:
In the sense of "we bail if it does exceed this limit" or "we can prove that it won't"? If the former, where? If the latter... maybe some more explanation?

Reply:
The latter.